docroot/modules/custom/uiowa_maui/src/Form/AcademicDatesForm.php

```diff
@@ -50,7 +50,7 @@ public function getFormId() {
   /**
    * {@inheritdoc}
    */
-  public function buildForm(array $form, FormStateInterface $form_state, $session_prefilter = NULL, $category_prefilter = NULL, $child_heading_size = NULL) {
+  public function buildForm(array $form, FormStateInterface $form_state, $session_prefilter = NULL, $category_prefilter = NULL, $child_heading_size = NULL, $items_to_display = NULL, $display_deadlines = NULL, $display_more_link = NULL, $display_more_text = NULL) {
```

Comment:
I think that it is not necessary for us to pass in the last two link-related arguments here. The more link could be rendered in `AcademicDatesBlock::build` right after the `$build['form']` element. For example, check if `$display_more_link` is set and if so, construct a link using https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21Element%21Link.php/class/Link/8.2.x.

Reply:
I can help out with this. :)

Reply:
Okay! I could definitely use help with this.